aws/lambda/python/s3_cleanup.py

```
@@ -0,0 +1,56 @@
+
+        events.delete_rule(Name=rule_name)
+        print(f"Deleted EventBridge rule: {rule_name}")
+
+        return {
+            "statusCode": 200,
+            "body": json.dumps(
+                f"Bucket {bucket_name} deleted. EventBridge rule removed."
+            ),
+        }
+
+    except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)